integration-test/helpers/pages.js

```diff
@@ -193,6 +199,29 @@ export function loginPage (page, server, opts = {}) {
                 await button.click()
             }
         },
+        /**
+         * @param {string} username
+         * @param {string} password
+         * @return {Promise<void>}
+         */
+        async assertBitwardenTooltipWorking (username, password) {
```

Comment:
These 2 methods for me, show the true power of this level of testing. It reads like a story and I can picture exactly supposed to be happening.